visual-js/visual-snapshots/src/api/visual-client.ts

```diff
@@ -0,0 +1,6 @@
+import { getApi, VisualConfig } from "@saucelabs/visual";
+
+export const initializeVisualApi = (params: VisualConfig) =>
+  getApi(params, {
+    userAgent: "visual-snapshots",
```

Comment:
You should probably add `PKG_VERSION` into this as well so it can be replaced by our version injection script. This will let us also determine what version is being used in the useragent.